Description: Fix libpam.so filename to /lib/libpam.so.0 in pam plugin
Author: Alberto Gonzalez Iniesta <agi@inittab.org>
Bug-Debian: http://bugs.debian.org/306335
Index: openvpn-2.2.1/plugin/auth-pam/auth-pam.c
===================================================================
--- openvpn-2.2.1.orig/plugin/auth-pam/auth-pam.c	2011-12-13 12:41:58.693936739 +0100
+++ openvpn-2.2.1/plugin/auth-pam/auth-pam.c	2011-12-13 12:42:00.146918574 +0100
@@ -688,7 +688,7 @@
   struct user_pass up;
   int command;
 #if DLOPEN_PAM
-  static const char pam_so[] = "libpam.so";
+  static const char pam_so[] = "libpam.so.0";
 #endif
 
   /*
